kms/README.md

```diff
@@ -20,8 +20,8 @@
 	- `authz-oathkeeper-proxy.DOMAIN`
 	- `ops-oathkeeper-proxy.DOMAIN`
 	- `vault-kms.DOMAIN`
-* Will deploy Sandbox KMS: authz and ops with [oathkeeper](https://github.com/ory/oathkeeper), and vault-kms pointing to an already provisioned CouchDB specified with `COUCHDB_URL`
-	- `make deploy-with-external-couchdb COUCHDB_URL=user:pass@couchdb-address:port`
+* Will deploy Sandbox KMS: authz and ops with [oathkeeper](https://github.com/ory/oathkeeper), and vault-kms pointing to an already provisioned MongoDB specified with `MONGODB_URL`
+	- `make deploy-with-external-mongodb MONGODB_URL=mongodb://user:pass@mongodb-address:27017`
```

Comment:
@rolsonquadras I noticed this `deploy-with-external-couchdb` make target mentioned in a few different README files, but I can't actually find it. Should I remove this, or perhaps update this somehow?

Reply:
Yeah, lets remove it if its not used @DRK3 (may be raise it as a separate PR)